src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -229,6 +318,16 @@ private function getErrorMessages(Exception $e): array
     {
         // @todo: all the constraint error messages are missing for now (see ProductConstraintException)
         return [
+            CannotDeleteProductException::class => [
+                CannotDeleteProductException::FAILED_DELETE => $this->trans(
+                    'An error occurred while deleting the object.',
+                    'Admin.Notifications.Error'
+                ),
+                CannotDeleteProductException::FAILED_BULK_DELETE => $this->trans(
+                    'An error occurred while deleting this selection.',
+                    'Admin.Notifications.Error'
+                ),
+            ],
```

Comment:
```suggestion
            CannotDeleteProductException::class => $this->trans(
                'An error occurred while deleting the object.',
                'Admin.Notifications.Error'
            ),
```

As explained in a previous comment this exception is only used for single deletion so no need to tell it apart from bulk delete